lib/cukejson/cucumberDataCollector.js

```diff
@@ -84,7 +84,7 @@ class CucumberDataCollector {
         this.stepResults[this.currentStep] = {
           status: statuses.FAILED,
           duration: this.timeTaken(),
-          exception: this.testError,
+          exception: this.testError.message,
```

Comment:
I'm not entirely sure how this will affect stuff. I just ressurected the old test suite a wrote a while ago and this was necessary to make it pass.